src/main/groovy/org/shipkit/internal/gradle/GithubPlugin.java

```diff
@@ -0,0 +1,16 @@
+package org.shipkit.internal.gradle;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.shipkit.internal.gradle.contributors.github.GithubContributorsPlugin;
+import org.shipkit.internal.gradle.java.GithubPomContributorsPlugin;
+
+
+public class GithubPlugin implements Plugin<Project> {
```

Comment:
When introducing new plugins / refactoring how existing plugins are used together, please update the Javadoc accordingly. See the Javadoc of surrounding plugins for reference. 1.0 is coming soon and we need the Javadoc to be comprehensive and accurate! :)